build/props.js

```diff
@@ -82,6 +84,32 @@ const figmatokensSYNC = { 'open-props': { ...figmatokens } }
 const FigmaTokensSync = fs.createWriteStream('../open-props.figma-tokens.sync.json')
 FigmaTokensSync.end(JSON.stringify(figmatokensSYNC, null, 2))
 
+if (!fs.existsSync('../dist'))
+  fs.mkdirSync('../dist')
```

Comment:
this fails on windows because of `/`.

this works:
```suggestion
const distPath = path.join(fileURLToPath(import.meta.url), '..', 'dist')

if (!fs.existsSync(distPath))
  fs.mkdirSync(distPath)
```

needs these imports:
```js
import path from 'node:path';
import { fileURLToPath } from 'node:url';
```

Reply:
actually i cannot get it to fail anymore. it's really weird.